### PR TITLE
Added A Project Description

### PR DESCRIPTION
**Pull request by Alex-is-Gonzalez**
-Added description to the top of the file to explain what Amplify does. 
 -Added a description to the video for the first video to clarify the purpose

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Amplify
-
+ Amplify is an open-source app created for users to take the initiative in being part of an actionable step in the efforts to protect against climate change. The user is able to choose a climate campaign, then using their zip code, they will be able to select a representative of their choice. The user then donates to have their letter sent out by Amplify.
```
Comment (paramsiddharth) [inline README.md:2]:
```suggestion
Amplify is an open-source app created for users to take the initiative in being part of an actionable step in the efforts to protect against climate change. The user is able to choose a climate campaign, then using their zip code, they will be able to select a representative of their choice. The user then donates to have their letter sent out by Amplify.
```

File: README.md [hunk 2]
```diff
@@ -29,15 +27,18 @@ The backend (API) code is stored in the `server/` directory.
...
 
 
 
-## Project setup
+# Project setup
+In order to get started, you can clone, download a ZIP, or fork this repository to work on your local machine. If you would like to get started with Codespaces instead, the video below will walk you through setting up your Codespace.
 
 https://user-images.githubusercontent.com/9143339/159093687-6fc90733-0599-445c-b08b-a6378d988e4b.mov
 
+<figcaption align = "center"><b>Codespaces Set Up</b></figcaption>
```
Comment (paramsiddharth) [inline README.md:40]: I don't think centring works here in GitHub's Markdown (check the preview), so you can get rid of the `figcaption` tag.

```suggestion
<b>Codespaces Set Up</b>
```

**Review by paramsiddharth — CHANGES_REQUESTED** [review]
.

**Review by paramsiddharth — APPROVED** [review]
Looks good to me! Well done, Alex. 🌼

**Comment by Alex-is-Gonzalez** [comment]
Thank you for your guidance, Param!

**Pull request merged**